VERSION

```diff
@@ -1 +1 @@
-0.60.0
+0.60.1
```

Comment:
https://github.com/status-im/status-go/pull/2033/files already has a version bump to `0.60.1` maybe set to `0.60.2`?

Reply:
Yep, merged, need to upgrade version